.github/CODEOWNERS

```
@@ -948,8 +948,7 @@
 # ServiceOwners:                                                   @davidemontanari @atastrophic @marcodalessandro @rohankhandelwal @riteshrao
 
 # PRLabel: %Informatica Data Management
-/sdk/informaticadatamanagement/Azure.ResourceManager
-.Informaticadatamanagement/                                        @almasak
+/sdk/informaticadatamanagement/Azure.ResourceManager.Informaticadatamanagement/ @almasak
```

```suggestion
/sdk/informaticadatamanagement/Azure.ResourceManager*/ @almasak
```

We can shorten the path and still achieve the desired effect.

---

Good suggestion, I have made the change.